simpeg/utils/logger.py

```diff
@@ -0,0 +1,23 @@
+"""
+Define global logger for SimPEG.
+"""
+
+import logging
+
+__all__ = ["LOGGER"]
+
+
+def _create_logger():
+    """
+    Create logger for SimPEG.
+    """
+    logger = logging.getLogger(__name__)
```

Comment:
This will make it look like every logged line comes from this file. It would make more sense to instantiate a logger for the simulation in my opinion.

Reply:
Make sense. What about a global logger for SimPEG?